CITATION.bib

```diff
@@ -0,0 +1,19 @@
+@software{qiskit-addon-sqd,
+  author = {
+    Abdullah Ash Saki
+    and Stefano Barison
+    and Bryce Fuller
+    and James R. Garrison
+    and Jennifer R. Glick
+    and Caleb Johnson
+    and Antonio Mezzacapo
+    and Javier Robledo-Moreno
+    and Max Rossmannek
+    and Paul Schweigert
+    and Iskandar Sitdikov
+    and Kevin J. Sung
+  },
+  title = {{Qiskit addon: sample-based quantum diagonalization}},
+  howpublished = {\url{https://github.com/Qiskit/qiskit-addon-sqd}},
+  year = {2024},
```

Comment:
I think you will be safer if you get rid of the comma, even though it seems to work here.

```suggestion
  year = {2024}
```